test/metabase/query_processor_test/split_part_test.clj

```diff
@@ -0,0 +1,67 @@
+(ns ^:mb/driver-tests metabase.query-processor-test.split-part-test
+  (:require
+   [clojure.string :as str]
+   [clojure.test :refer :all]
+   [metabase.lib.core :as lib]
+   [metabase.lib.metadata :as lib.metadata]
+   [metabase.query-processor :as qp]
+   [metabase.test :as mt]))
+
+(deftest ^:parallel split-part-test
```

Comment:
@bshepherdson I believe these tests will run for all drivers with :split-part. Is that correct? Even the ones in the `modules/` dir?

Reply:
Yes, we run tests in CI against all the 1P drivers, ie. those in the `modules/` directory.